src/components/dashboard/sidebar/LearnerCreditSummaryCard.jsx

```diff
@@ -52,94 +53,114 @@ const conditionallyRenderCardBadge = ({
   badgeVariant,
   intl,
 }) => {
+  let renderedBadge;
+
+  // If the disable learner credit expiration messaging is enabled, default to an active badge
   if (status === BUDGET_STATUSES.expiring && disableExpiryMessagingForLearnerCredit) {
-    return null;
```

Comment:
Nit: since this is no longer conditionally rendering the card badge, maybe change the name of the function to match the new behavior?  perhaps "getRenderedBadge()"?